osu.Game/Overlays/Profile/Sections/PaginatedContainer.cs

```diff
@@ -80,13 +84,57 @@ private void load(IAPIProvider api, RulesetStore rulesets)
 
         private void onUserChanged(ValueChangedEvent<User> e)
         {
+            loadCancellation?.Cancel();
```

Comment:
because this is changing a sizeable amount of important logic, please split this into its own PR.

Reply:
Sure